x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -22,17 +23,20 @@ import { EnginesListTable } from './components/tables/engines_table';
 import { DeleteEngineModal } from './delete_engine_modal';
 import { EnginesListLogic } from './engines_list_logic';
 
+const ENGINES_SEARCH_THROTTLE_MS = 500;
```

Comment:
we may want to put this in shared constants.

Reply:
Why did we choose 500? It looks like [other apps](https://github.com/elastic/kibana/blob/364ab545a4b34d0d498449af9a8959ad32539b5b/x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/logs.tsx#L102) use 1000?

Reply:
@kderusso i just pulled it out of thin air